packages/blunderbuss/test/blunderbuss.ts

```diff
@@ -93,6 +96,32 @@ describe('Blunderbuss', () => {
     nock.cleanAll();
   });
 
+  describe('scheduler handler', () => {
+    it('calls syncLabels', async () => {
+      await probot.receive({
+        name: 'schedule.repository' as '*',
+        payload: {
+          repository: {
+            name: 'testRepo',
+            owner: {
+              login: 'testOwner',
+            },
+          },
+          organization: {
+            login: 'googleapis',
+          },
+        },
+        id: 'abc123',
+      });
+      syncLabelsStub.calledOnceWith(
+        sinon.match.instanceOf(Octokit),
+        'testOwner',
```

Comment:
It seems like calledOnceWith is broken?

I think I need to call `getCalls()` or something.